<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>72</y>
      <width>991</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>371</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>url：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_url"/>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>价格：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_price"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_dump_excel">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导出为excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dump_json">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导出为json</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>532</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string> 系统：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_sys"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> 员工：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_user"/>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QPushButton" name="btn_search">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string> 币种：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_coin"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string> 用途：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_logic"/>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btn_reinvest">
       <property name="text">
        <string>复投</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="btn_release">
       <property name="text">
        <string>解套</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>一键加仓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_reverse">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>反向加仓</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen_json"/>
    <addaction name="actionopen_excel"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="actionasset"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen_json">
   <property name="text">
    <string>打开json文件</string>
   </property>
  </action>
  <action name="actionopen_excel">
   <property name="text">
    <string>打开excel文件</string>
   </property>
  </action>
  <action name="actionasset">
   <property name="text">
    <string>资产</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
